README.md

```diff
@@ -61,6 +61,11 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
   - `internalConsole` - VS Code debug console (input stream not supported).
   - `integratedTerminal` - VS Code integrated terminal.
   - `externalTerminal` - External terminal that can be configured in user settings.
+- `stepFilters` - Customize the filters to skip the specified methods and classes when stepping.
+  - `classNameFilters` - Skip these classes when stepping. Each filter could be fully qualified class name or regular expression that begins with '\*' or ends with '\*'. Example: [\"java.lang.ClassLoader\", \"java.*\"].
```

Comment:
Skip the specified classes when stepping. Class names should be fully qualified. Wildcard is supported.